src/control/server/ctl_harness.go

```diff
@@ -85,7 +84,8 @@ func (hc *harnessClient) prepareRequest(req RemoteHarnessReq) (*mgmtpb.RanksReq,
 	return &mgmtpb.RanksReq{Ranks: req.Ranks, Timeout: req.Timeout, Force: req.Force}, nil
 }
 
-func (hc *harnessClient) processResponse(rpcResp *mgmtpb.RanksResp) (system.MemberResults, error) {
+// processRPC takes a RanksResp proto message and converts to system.MemberResults.
+func (hc *harnessClient) processRPC(rpcResp *mgmtpb.RanksResp) (system.MemberResults, error) {
```

Comment:
I'm curious about the motivation behind the name changes here. The other names seemed clearer to me. The purpose of ```processRPC()``` is a bit vague without context (is it handling a RPC?), whereas ```processResponse()``` is pretty clear.

They're not exported methods so this doesn't bother me enough to block on.

Reply:
thought it made more sense at the time , but I will change back